hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -336,6 +338,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
     switch (result) {
     case SUCCESS:
+      omMetrics.incNumKeys(numMissingParents);
```

Comment:
We need to add incNumCreateFileFails for the failure case below?

Reply:
This is already taken care 
```
    } catch (IOException ex) {
      result = Result.FAILURE;
      exception = ex;
      omMetrics.incNumCreateFileFails();
```

Is this what you are looking for?

Reply:
yes. Thanks @bharatviswa504